### PR TITLE
[CS-3962] - Remove dead pkgs and improve wallet init

### PR DESCRIPTION
**Pull request by danibonilha**
### Description

As part of the heating issue, it was identified a bunch of unnecessary subscriptions for swaping, which was removed on this PR, to improve the app performance some init methods were changed in order to init faster.

- Removes uniwasp/swap features
- Updates Loading to use same component across application (This is one is a bit out of scope, but the loadingOverlay update makes the UI faster )
- Removes unused iOS internal packages
- Migrates coingecko call to fallbackback explorer and caches it, in order to improve wallet init
- Removes unnecessary data clean up (we were cleaning up, loading, then cleaning up again theen loading the final piece).
- Improves safes fetch, no need to wait for wallet to be ready now, as we already have conversionRates, refer to #528 for more context why we needed that before.
- Removes unused  charts reducer

The start of the application has improved overall from 11-12 to  8 seconds.

### CODE REVIEW

File: src/redux/fallbackExplorer.js [hunk 1]
```diff
@@ -95,15 +96,31 @@ const fetchCoingeckoIds = async (network, coingeckoCoins) => {
...
+    const request = await fetch(
+      'https://api.coingecko.com/api/v3/coins/list?include_platform=true&asset_platform_id=ethereum'
+    );
+    const coingeckoCoins = await request.json();
+
+    return coingeckoCoins;
+  } catch (error) {
+    return coingeckoIdsFallback;
+  }
+};
```
Comment (danibonilha) [inline src/redux/fallbackExplorer.js:112]: The TODO initially it said to move rtk, which made sense at the time, but now that is only user here, it makes more sense to scope this call within the fallbackExplorer. This is just a list of available coins, so it's ok to fetch just once in a while

File: cardstack/src/components/LoadingOverlay/LoadingOverlay.tsx [hunk 2]
```diff
@@ -1,32 +1,17 @@
...
 
-const transition = (
-  <Transition.Sequence>
-    <Transition.Out durationMs={500} interpolation="easeOut" type="fade" />
-    <Transition.Change durationMs={500} interpolation="easeOut" />
-    <Transition.In durationMs={500} interpolation="easeOut" type="fade" />
-  </Transition.Sequence>
-);
-
 const OverlyStyle = {
```
Comment (oidoug) [inline cardstack/src/components/LoadingOverlay/LoadingOverlay.tsx]: Possibly old typo, OverlyStyle => overlayStyle

File: cardstack/src/components/LoadingOverlay/LoadingOverlay.tsx [hunk 3]
```diff
@@ -42,37 +27,29 @@ const OverlyStyle = {
...
+}: {
+  title: string;
+  subTitle?: string;
+}) => (
+  <CenteredContainer style={styles.overlayWrapper}>
+    <CenteredContainer
+      backgroundColor="white"
+      borderColor="whiteOverlay"
+      shadowColor="overlay"
+      {...OverlyStyle}
```
Comment (oidoug) [inline cardstack/src/components/LoadingOverlay/LoadingOverlay.tsx]: ditto

File: cardstack/src/components/LoadingOverlay/LoadingOverlay.tsx [hunk 4]
```diff
@@ -42,37 +27,29 @@ const OverlyStyle = {
...
-  ({ title, subTitle }: { title: string; subTitle: string }) => {
-    return (
-      <Transitioning.View transition={transition} style={styles.overlayWrapper}>
-        <CenteredContainer
-          backgroundColor="white"
-          borderColor="whiteOverlay"
-          shadowColor="overlay"
-          {...OverlyStyle}
-        >
-          {Device.isAndroid ? (
```
Comment (oidoug) [inline cardstack/src/components/LoadingOverlay/LoadingOverlay.tsx:55]: 👏

File: cardstack/src/hooks/merchant/usePaymentMerchantUniversalLink.ts [hunk 5]
```diff
@@ -59,13 +58,11 @@ export const usePaymentMerchantUniversalLink = () => {
...
   const [infoDID, setInfoDID] = useState<string | undefined>();
 
-  const walletReady = useRainbowSelector(state => state.appState.walletReady);
-
   const { isLoading = true, prepaidCards } = useGetPrepaidCardsQuery(
     { accountAddress, nativeCurrency },
     {
       refetchOnMountOrArgChange: 60,
-      skip: isLayer1(accountNetwork) || !accountAddress || !walletReady,
+      skip: isLayer1(accountNetwork) || !accountAddress,
```
Comment (oidoug) [inline cardstack/src/hooks/merchant/usePaymentMerchantUniversalLink.ts:65]: Can you elaborate on this change? Does this mean the Wallet doesn't need to be ready before fetching safes?

File: src/redux/fallbackExplorer.js [hunk 6]
```diff
@@ -95,15 +96,31 @@ const fetchCoingeckoIds = async (network, coingeckoCoins) => {
...
+const findAssetsToWatch = async (address, latestTxBlockNumber, dispatch) => {
   // 1 - Discover the list of tokens for the address
   const network = store.getState().settings.network;
-  const coingeckoIds = await fetchCoingeckoIds(network, coingeckoCoins);
+
+  coingeckoCoinsCache.current =
+    coingeckoCoinsCache.current || (await fetchCoingeckoCoins());
```
Comment (oidoug) [inline src/redux/fallbackExplorer.js:118]: Note: You mentioned above that fetching once in a while was ok, but by saving the fetch result in this global variable you'll only refresh values on app re-runs.

**Review by oidoug — APPROVED** [review]
LGTM! Impressive work. I left some notes for your appreciation.

File: cardstack/src/hooks/merchant/usePaymentMerchantUniversalLink.ts (hunk 5 above)
Comment (danibonilha) [reply cardstack/src/hooks/merchant/usePaymentMerchantUniversalLink.ts:65]: Yep, we already have the exchange rates now, this old implementation was add on this PR #528, you can find more context there.

File: src/redux/fallbackExplorer.js (hunk 6 above)
Comment (danibonilha) [reply src/redux/fallbackExplorer.js:118]: Even though i'm not sure about the coins update frequency, on app re-runs should be enough to get new coins, if any is added.

**Pull request merged**